pkgs/racket-doc/scribblings/reference/units.scrbl

```diff
@@ -759,11 +791,21 @@ superset of the export signatures listed in the unit contract. Additionally,
 the unit value must declare initialization dependencies that are a subset of
 those specified in the unit contract. Any identifier which is not listed
 for a given signature is left alone. Variables used in a given
-@racket[contract] expression first refer to other variables in the same
-signature, and then to the context of the @racket[unit/c] expression.
+@racket[contract] expression first refer to other variables in any of the
+listed signatures, and then to the context of the @racket[unit/c] expression.
 If a body contract is specified then the result of invoking the unit value
-is wrapped with the given contract, if no body contract is supplied then
-no wrapping occurs when the unit value is invoked.}
+is wrapped with the given contract, otherwise the values are returned as-is.
+
+@history[
+ #:changed "8.8.0.2" @elem{Changed @racket[sig-spec-block] to allow arbitrary
+                           @racket[tagged-sig-spec]s instead of only allowing
+                           @racket[tagged-sig-id]s.}
+ #:changed "8.8.0.2" @elem{Made bindings from @emph{all} signatures visible in
+                           the scope of each @racket[contract] expression
+                           instead of only the bindings from the same signature.
+                           Additionally, contracts on signature bindings are
+                           enforced for uses that appear in the @racket[contract]
+                           expressions themselves.}]}
 
```

Comment:
I'd normally put both changes in the same `#:changed`, so doing that is probably more consistent with other documentation, but I don't feel strongly that it has to be that way.

Reply:
I certainly don’t have a strong opinion, so I’ve merged them together into the same use of `#:changed`.